### ISSUE TITLE
Add keyboard + screen reader accessiblity to guppy

### ISSUE THREAD

**Issue by Patel-Muhammad** [daniel3735928559/guppy#221]
Currently, guppy does not have any mechanism to provide keyboard/screenreader accessibility support.

**Expected Behavior**:

Chromevox should read the entire text of the label instead of each letter. Also with chromevox user should be able to enter data into the textbox

**Videos**:

https://user-images.githubusercontent.com/101634267/230885744-3802fe4c-198d-4c2c-98f1-5e81192365a2.mp4
